src/main/java/net/neoforged/neoforge/common/data/LanguageProvider.java

```diff
@@ -114,11 +115,15 @@ public void addTag(Supplier<? extends TagKey<?>> key, String name) {
     }
 
     public void add(TagKey<?> tagKey, String name) {
-        add(Tags.getTagTranslationKey(tagKey), name);
+        add(net.neoforged.neoforge.common.Tags.getTagTranslationKey(tagKey), name);
     }
 
     public void add(String key, String value) {
         if (data.put(key, value) != null)
             throw new IllegalStateException("Duplicate translation key " + key);
     }
+
+    public void addDimension(ResourceKey<Level> dimension, String value) {
+        add(dimension.location().toLanguageKey(net.neoforged.neoforge.common.extensions.ILevelExtension.TRANSLATION_PREFIX), value);
```

Comment:
```suggestion
        add(dimension.location().toLanguageKey(ILevelExtension.TRANSLATION_PREFIX), value);
```
Add an import instead of using an FQN

Reply:
Noticed this right after I clicked the push button, should be fixed now

Reply:
Wait no... My mistake again, the name `LanguageProvider` made me think its a vanilla file, so I FQN'd but its a neo file 🤦.

This is what I get for doing a "quick rebase" immediately after waking up.